pkg/k8shandler/elasticsearch_test.go

```diff
@@ -31,12 +63,268 @@ func TestHeaderGenWithToken(t *testing.T) {
 	}
 }
 
-func TestHeaderGenWithNoToken(t *testing.T) {
+/*func TestHeaderGenWithNoToken(t *testing.T) {
 	tokenFile := "../../test/files/errorToken"
```

Comment:
@jcantrill not sure if i'm missing a fix but this file doesn't seem to be here on my branch